### ISSUE TITLE
No Support for Dreame L10 Ultra 

### ISSUE THREAD

**Issue by Janek2002** [Tasshack/dreame-vacuum#168]
Unfortunately, there is no support for the Dreame L10 Ultra for this integration.
   Is there a specific reason for it?
  Could it be because the robotic vacuum cleaner does not have a camera?

**Comment by tjuanma** [comment]
I'm looking for the L10 Ultra integration too.
It's something that will happend? :D

**Comment by sgozz** [comment]
I’m waiting too :)

**Comment by tjuanma** [comment]
> I’m waiting too :)

You been able to connect L10 ultra to xiaomi miHome app? I'm just able to do it by dreame app.

**Comment by sgozz** [comment]
Nope, only with Dreame App. I think we will have to wait for Dreame login integration to be finished.

**Comment by tjuanma** [comment]
> Nope, only with Dreame App. I think we will have to wait for Dreame login integration to be finished.

Yes...i think that is the main reason

**Comment by Janek2002** [comment]
So now, just wait and hope that it will soon be possible to sign up with the Dreame app?
